frontend/src/metabase/admin/components/Page.info.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import { PageHeader, PageTools, PageActions, PageTabs } from "./Page";
+
+import Icon from "metabase/components/Icon";
+import Radio from "metabase/components/Radio";
+import Subhead from "metabase/components/type/Subhead";
+
+export const component = PageHeader;
+export const category = "admin";
+export const description = `Layout recipes for admin page headers`;
+
+export const examples = {
+  "": (
+    <PageHeader>
+      <Subhead>Database listing page</Subhead>
+    </PageHeader>
+  ),
+  withAction: (
+    <PageHeader>
+      <PageTools>
```

Comment:
I don't love this name but I couldn't think of anything better at the time for a section that's supposed to act as a wrapper in the case you have actions, or actions and tabs.